src/overlay/template.jsx

```diff
@@ -50,6 +50,14 @@ export type OverlayProps = {|
   fullScreen?: boolean,
 |};
 
+function showAlert(message) {
+  return new Promise((resolve) => {
```

Comment:
for now, until we can figure out how to migrate off of it, this should be `ZalgoPromise`
you can import it like:
```
import { ZalgoPromise } from "@krakenjs/zalgo-promise/src";
```

```suggestion
  return new ZalgoPromise((resolve) => {
```

Reply:
Got it! Thanks @wsbrunson